Jellyfin.Plugin.PlaybackReporting/Pages/duration_histogram_report.js

```diff
@@ -201,7 +201,9 @@ define(['libraryMenu'], function (libraryMenu) {
                             }
                         }
 
-                        var days = parseInt(weeks.value) * 7;
+                       var days = parseInt(weeks.value) * 7;
+                       if (days == -7) days = 18250;
```

Comment:
I know this is from @dmitrylyzo but please document what this magic number is for

Reply:
i seem to have removed the existing comment, will go ahead and add them in.

Reply:
And fix indent please.